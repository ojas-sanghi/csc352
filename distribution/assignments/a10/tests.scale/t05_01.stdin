11558
52150
22839
36477
69290
