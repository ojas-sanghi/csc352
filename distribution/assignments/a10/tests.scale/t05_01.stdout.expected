11558 (audit=1)
scaled by 1: 11558 (audit=1)
scaled by 2: 23116 (audit=1)
scaled by 3: 69348 (audit=1)
scaled by 4: 277392 (audit=1)
scaled by 5: 1386960 (audit=1)
scaled by 6: 8321760 (audit=1)
scaled by 7: 58252320 (audit=1)
scaled by 8: 466018560 (audit=1)
scaled by 9: 4194167040 (audit=1)
52150 (audit=1)
scaled by 1: 52150 (audit=1)
scaled by 2: 104300 (audit=1)
scaled by 3: 312900 (audit=1)
scaled by 4: 1251600 (audit=1)
scaled by 5: 6258000 (audit=1)
scaled by 6: 37548000 (audit=1)
scaled by 7: 262836000 (audit=1)
scaled by 8: 2102688000 (audit=1)
scaled by 9: 18924192000 (audit=1)
22839 (audit=1)
scaled by 1: 22839 (audit=1)
scaled by 2: 45678 (audit=1)
scaled by 3: 137034 (audit=1)
scaled by 4: 548136 (audit=1)
scaled by 5: 2740680 (audit=1)
scaled by 6: 16444080 (audit=1)
scaled by 7: 115108560 (audit=1)
scaled by 8: 920868480 (audit=1)
scaled by 9: 8287816320 (audit=1)
36477 (audit=1)
scaled by 1: 36477 (audit=1)
scaled by 2: 72954 (audit=1)
scaled by 3: 218862 (audit=1)
scaled by 4: 875448 (audit=1)
scaled by 5: 4377240 (audit=1)
scaled by 6: 26263440 (audit=1)
scaled by 7: 183844080 (audit=1)
scaled by 8: 1470752640 (audit=1)
scaled by 9: 13236773760 (audit=1)
69290 (audit=1)
scaled by 1: 69290 (audit=1)
scaled by 2: 138580 (audit=1)
scaled by 3: 415740 (audit=1)
scaled by 4: 1662960 (audit=1)
scaled by 5: 8314800 (audit=1)
scaled by 6: 49888800 (audit=1)
scaled by 7: 349221600 (audit=1)
scaled by 8: 2793772800 (audit=1)
scaled by 9: 25143955200 (audit=1)
